### PR TITLE
[MFTF] Cover Used In Media Gallery filter with MFTF

### LINKED ISSUE #1567: [MFTF] Cover Used In Media Gallery filter with MFTF tests
### Preconditions (*)
1. Installed Magento with Adobe Stock Integration
2. Configured  integration in Stores -> Configuration -> System -> Adobe Stock Integration fieldset

### Steps to reproduce (*)
Admin user is logged in to admin panel
Go to Stores - Configuration under Settings section 
Assert that Configuration window is opened
User expands Advanced tab
User click System
User expands Enhanced Media Gallery tab
User selects Yes from Enabled dropdown
User clicks Save config button
Assert that the configuration has been saved 
User has several images in Media Gallery
User creates a new Page and adds an image to its Content
Navigate to cms/page page
Click Add New Page button
Expand Content fieldset
Click Show/Hide Editor button
Click Insert Image ... button
User expand 'Filters'
Assert that 'Show Images Used In' drop down filter is displayed with the default value - "Select..."
User clicks on 'Show Images Used In' drop down
Assert that the following check boxes are available: 'Pages', 'Categories', 'Blocks', 'Products'
User checks the 'Pages' checkbox 
Assert that the 'Pages' block appeared in 'Show images Used In' filter field
User clicks Done button
Assert that the drop down is minimized
User click 'Apply Filters'
Assert that the "Active filters: Show Images Used In: Page" is applied
Assert that the image that was previously added to a Page is displayed
Clear filters
Assert that all the images from Media Gallery Grid are displayed

### Note (*)
Story - https://github.com/magento/adobe-stock-integration/issues/1291
Issue - https://github.com/magento/adobe-stock-integration/issues/1462
PR - https://github.com/magento/adobe-stock-integration/pull/1531
Test on Cucumber - https://studio.cucumber.io/projects/131313/test-plan/folders/1320712/scenarios/4930844

### PR DESCRIPTION
**Pull request by Nazar65**
### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1567:[MFTF] Cover Used In Media Gallery filter with MFTF
2. ...

### Manual testing scenarios (*)
functional tests are green

### CODE REVIEW

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
@magento run all tests

File: MediaGalleryUi/Test/Mftf/Section/AdminEnhancedMediaGalleryFiltersSection.xml [hunk 1]
```diff
@@ -13,5 +13,8 @@
         <element name="applyFilters" type="button" selector="//div[@class='media-gallery-container']//button[@data-action='grid-filter-apply']"/>
         <element name="activeFilter" type="text" selector="//div[@class='media-gallery-container']//div[@class='admin__current-filters-list-wrap']//span[contains( ., '{{filter}}')]" parameterized="true"/>
         <element name="activeFilterPlaceholder" type="text" selector="//div[@class='media-gallery-container']//div[@class='admin__current-filters-list-wrap']"/>
+        <element name="usedInSelectDropdown" type="text" selector="//div[@class='admin__action-multiselect-text' and text()='Select...']"/>
```
Comment (sivaschenko) [inline MediaGalleryUi/Test/Mftf/Section/AdminEnhancedMediaGalleryFiltersSection.xml]: The selectors should be more specific, we are going to have more than one multiselect in the filters section

File: MediaGalleryUi/Test/Mftf/Test/AdminEnhancedMediaGalleryVerifyUsedInFilterTest.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        <actionGroup ref="AdminMediaGalleryClickImageInGridActionGroup" stepKey="selectCategoryImageInGrid">
+            <argument name="imageName" value="{{ImageUpload.file}}"/>
+        </actionGroup>
+        <actionGroup ref="AdminMediaGalleryClickAddSelectedActionGroup" stepKey="clickAddSelectedContentImage"/>
+        <actionGroup ref="AdminMediaGalleryClickOkButtonTinyMce4ActionGroup" stepKey="clickOkButton"/>
+        <actionGroup ref="AdminSaveCategoryActionGroup" stepKey="saveCategory"/>
+        <actionGroup ref="AdminOpenMediaGalleryTinyMce4ActionGroup" stepKey="openMediaGalleryFromWysiwygToFilterIMage"/>
+
+        <actionGroup ref="AdminEnhancedMediaGalleryExpandFilterActionGroup" stepKey="expandFilters"/>
+        <actionGroup ref="AdminMediaGalleryApplyUsedInFIlterActionGroup" stepKey="applyUsedInCategoryFilter">
```
Comment (sivaschenko) [inline MediaGalleryUi/Test/Mftf/Test/AdminEnhancedMediaGalleryVerifyUsedInFilterTest.xml]:
```suggestion
        <actionGroup ref="AdminMediaGalleryApplyUsedInFilterActionGroup" stepKey="applyUsedInCategoryFilter">
```

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the PR @Nazar65 , please take a look at my comments

File: MediaGalleryUi/Test/Mftf/Section/AdminEnhancedMediaGalleryFiltersSection.xml (hunk 1 above)
Comment (Nazar65) [reply MediaGalleryUi/Test/Mftf/Section/AdminEnhancedMediaGalleryFiltersSection.xml]: ok

**Comment by Nazar65** [comment]
@magento run all tests

**Comment by Nazar65** [comment]
Hi @sivaschenko thanks for the review, done :heavy_check_mark:

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**